src/components/WwwFrame/LendMenu/LendListMenu.vue

```diff
@@ -4,12 +4,9 @@
 			v-if="showMGUpsellLink"
 			to="/monthlygood"
 			@click.native="trackMgLinkClick"
-			:class="{ 'hover:tw-no-underline': newMgEntrypoint}"
 		>
-			<!-- NEW MG ENTRYPOINT CORE-641 -->
-			<monthly-good-entrypoint v-if="newMgEntrypoint" />
```

Comment:
There's still a couple instances of `newMgEntrypoint` than can be removed from this component.